Thumbprint/Targets/Thumbprint/Components/Avatar.swift

```diff
@@ -7,10 +7,21 @@ import UIKit
 
 public final class Avatar: UIView {
     public struct Size: Equatable {
+        public struct BadgeOffset: Equatable {
```

Comment:
Do we need to use a custom type for something that's functionally equivalent to a `CGPoint`?

Reply:
If you want to get particular about semantics maybe use a `CGVector` instead

Reply:
I considered that- I thought this was more clear in terms of which field corresponds to top offset vs right offset, but I can switch to CGPoint if you think x=>right, y=>top is clear enough

Reply:
Oh I like that, didn't know about `CGVector`

Reply:
I think `CGVector` + the comment mentioning it's top + right (trailing?) offset is clear enough.